fuzz/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "hex-fuzz"
-edition = "2018"
+edition = "2021"
+rust-version = "1.63.0"
```

Comment:
I wish we could pull these from workspace already... It's just one version away.